internal/publisher/pubsub_publisher.go

```diff
@@ -0,0 +1,82 @@
+package publisher
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/KyberNetwork/evmlistener/pkg/common"
+	"github.com/KyberNetwork/evmlistener/pkg/types"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+type PubsubPublisher struct {
```

Comment:
Should rename it to specify the purpose of data team, we may have another pubsub publisher that will encoding messages by protobuf but send message that include inverted blocks and new blocks.